frontend/common/src/defi/balancer/index.ts

```diff
@@ -1,4 +1,4 @@
-import { type default as BigNumber } from 'bignumber.js';
+import { type BigNumber } from 'bignumber.js';
```

Comment:
This seems like accidental do you think we can change this to import from the index re-export?